### PR TITLE
Add ogg validation in the editor

### LINKED ISSUE #6686: Add `ogg` files validation in the editor
**Task (REQUIRED):**
[Currently](https://github.com/defold/defold/pull/6554) bob.jar validates `ogg` files using `oggz-validate` tool. 
It would be nice if editor will do the same when user add new `ogg` file into component to catch potential problems earlier.

**Expected outcome (REQUIRED):**
If user adds an `ogg` file which isn't valid, the editor should notify about it.

### PR DESCRIPTION
**Pull request by vlaaad**
Technical notes:
On a big project I tried, total validation time was 1.5 sec over 212 files (where the total compilation time is 133 sec), i.e. the build got around 1% slower, which I think is fine. I verified on macOS, Windows and Linux that it works.

Fixes #6686

### CODE REVIEW

File: editor/src/clj/editor/sound.clj [hunk 1]
```diff
@@ -101,13 +129,12 @@
...
-                         {:path [:pan]
+                        {:path [:pan]
                          :label "Pan"
                          :type :number}
-                         {:path [:speed]
+                        {:path [:speed]
                          :label "Speed"
-                         :type :number}
-                         ]}]
+                         :type :number}]}]
```
Comment (vlaaad) [inline editor/src/clj/editor/sound.clj:137]: Indentation

File: editor/src/clj/editor/sound.clj [hunk 2]
```diff
@@ -176,16 +203,16 @@
                                             [:resource :sound-resource]
                                             [:build-targets :dep-build-targets])))
             (dynamic error (g/fnk [_node-id sound]
-                                  (or (validation/prop-error :info _node-id :sound validation/prop-nil? sound "Sound")
-                                      (validation/prop-error :fatal _node-id :sound validation/prop-resource-not-exists? sound "Sound"))))
+                             (or (validation/prop-error :info _node-id :sound validation/prop-nil? sound "Sound")
+                                 (validation/prop-error :fatal _node-id :sound validation/prop-resource-not-exists? sound "Sound"))))
```
Comment (vlaaad) [inline editor/src/clj/editor/sound.clj:207]: Indentation

File: editor/src/clj/editor/sound.clj [hunk 3]
```diff
@@ -176,16 +203,16 @@
...
             (dynamic edit-type (g/constantly {:type resource/Resource :ext supported-audio-formats})))
 
   (property looping g/Bool (default false))
   (property loopcount g/Int
             (value (g/fnk [looping loopcount]
                      (if (not looping) 0 loopcount)))
             (dynamic error (g/fnk [_node-id loopcount]
-                             (validation/prop-error :fatal _node-id :loopcount (partial validation/prop-outside-range? [0 127]) loopcount "Loopcount" )))
+                             (validation/prop-error :fatal _node-id :loopcount (partial validation/prop-outside-range? [0 127]) loopcount "Loopcount")))
```
Comment (vlaaad) [inline editor/src/clj/editor/sound.clj:215]: Indentation

**Review by matgis — APPROVED** [review]
Nicely done! LGTM!

**Pull request merged**